src/alignment.hpp

```diff
@@ -98,6 +98,7 @@ Alignment reverse_complement_alignment(const Alignment& aln, const function<int6
 vector<Alignment> reverse_complement_alignments(const vector<Alignment>& alns, const function<int64_t(int64_t)>& node_length);
 int softclip_start(const Alignment& alignment);
 int softclip_end(const Alignment& alignment);
+int query_overlap(const Alignment& aln1, const Alignment& aln2);
```

Comment:
This is the length of the overlap between the two alignments assuming they are both alignments of the same query.